tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+skipsdist = True
+envlist = unit, flake8, integration-synapse
+
+[testenv:integration-synapse]
+basepython = python3
+commands = /bin/bash -c '{envpython} -m pytest -v test/integration/synapse.dbtspec'
```

Comment:
Here you refer to it as `test/integration/synapse.dbtspec`, but the file is named `test/integration/sqlserver.dbtspec`. I think that might be it?